src/test/java/space/obminyashka/items_exchange/util/UserDtoCreatingUtil.java

```diff
@@ -17,17 +18,14 @@
     public static final List<ChildDto> EXISTED_CHILDREN = List.of(
             new ChildDto(1L, Gender.MALE, LocalDate.of(2019, 1, 1)),
             new ChildDto(2L, Gender.FEMALE, LocalDate.of(2019, 2, 2)));
-    public static final List<ChildDto> NEW_CHILDREN = List.of(
-            new ChildDto(0, Gender.MALE, LocalDate.of(2019, 3, 3)));
     public static final Set<PhoneDto> NEW_PHONES = Set.of(
             new PhoneDto(0, "381234567890", true));
-    public static final String NEW_USERNAME = "newUsername123";
     public static final String NEW_VALID_EMAIL = "new.admin@gmail.com";
     public static final String NEW_INVALID_DUPLICATE_EMAIL = "test@test.com";
     public static final String OLD_USER_VALID_EMAIL = "user@gmail.com";
     public static final String OLD_ADMIN_VALID_EMAIL = "admin@gmail.com";
-    public static final LocalDateTime NEW_LAST_ONLINE_TIME = LocalDateTime.now();
     public static final String NEW_VALID_NAME_WITH_APOSTROPHE = "Мар'яна";
+    public static final String NEW_VALID_NAME = "Мар'яна";
```

Comment:
Why didn't you use `NEW_VALID_NAME_WITH_APOSTROPHE` instead of new one?

Reply:
will be removed